src/main/java/com/redhat/labs/lodestar/models/pagination/Page.java

```diff
@@ -0,0 +1,132 @@
+package com.redhat.labs.lodestar.models.pagination;
```

Comment:
Should PagedResults class be in this package now? Also if moving can we downgrade ```LOGGER.debug("X-Total-Pages header is missing");``` to trace I don't think that header is comming back anytime soon.

Reply:
moved to pagination package and changed log leve.